### PR TITLE
Cutting scope is part of project leadership

### PR DESCRIPTION
**Pull request by rosenhouse**
Implementing #37

### CODE REVIEW

**Review by aramprice — APPROVED** [review]
this looks good to me. I like the addition of this as a skill and I think the arc of growth across the skill levels looks good.

File: project-leadership.md [hunk 1]
```diff
@@ -29,6 +30,8 @@ One example of success in this area: You are anchoring your team. You’ve been
   <li>Volunteers for facilitating team meetings</li>
 
   <li>Discusses team “things” with their anchor (eg: team process, team health/morale, product, architecture)</li>
+
+  <li>Is comfortable letting go of a priority that they have worked on</li>
```
Comment (aramprice) [inline project-leadership.md:27]: There might be a more concrete way of expressing this, or perhaps color to be added:
> Is comfortable when a thing they have worked on is no longer a priority

... maybe this idea is better as an "explainer" for this?

File: project-leadership.md [hunk 2]
```diff
@@ -46,6 +49,8 @@ One example of success in this area: You are anchoring your team. You’ve been
   <li>Can provide technical leadership on their team, and are able to share their technical vision with team members</li>
 
   <li>Able to anchor small/medium-sized teams with well-defined product priorities and technical direction</li>
+
+  <li>Recognizes when their pair or team would benefit from cutting scope, and builds consensus for it</li>
```
Comment (rnandi) [inline project-leadership.md:48]: How about something like "Recognizes when and helps their pair (or small team) feel comfortable cutting scope from a large story or small track"?

File: project-leadership.md (hunk 1 above)
Comment (rosenhouse) [reply project-leadership.md:27]: Actually, we're just going to drop this one.  It is the absence of a reaction, and isn't really worth trying to track.

**Review by rnandi — APPROVED** [review]

**Pull request merged**